src/aws-compile-servicecatalog.js

```diff
@@ -201,7 +198,11 @@ class AwsCompileServiceCatalog {
       || this.serverless.service.provider.runtime
       || 'nodejs4.3';
 
-    setProvisioningParamValue(this.getParameterName('handler'), functionObject.handler);
+    if (functionObject.handler) {
+      setProvisioningParamValue(this.getParameterName('handler'), functionObject.handler);
+    } else {
+      setProvisioningParamValue(this.getParameterName('image'), functionObject.image);
```

Comment:
This is going to require work in ServiceCatalog as well. The templates don't currently accept an Image parameter. You'll want to (1) update the sample templates included in this repo and (2) update our Service Catalog products (I'll Slack you the URL).

Reply:
Thank you!